lib/amorail/entities/contact.rb

```diff
@@ -1,43 +1,36 @@
 module Amorail
   class AmoContact < Amorail::AmoEntity
-    set_amo_name "contacts"
+    set_amo_name 'contacts'
 
     attr_accessor :name, :company_name, :linked_leads_id,
                   :email, :phone, :job_position
 
     validates :name, presence: true
 
-    def create_params
+    def params
       {
-        request: {
-          contacts: {
-            add: [
-              {
-                name: name,
-                date_create: to_timestamp(date_create),
-                last_modified: to_timestamp(last_modified),
-                request_id: request_id,
-                responsible_user_id: responsible_user_id,
-                company_name: company_name,
-                linked_leads_id: [linked_leads_id],
-                custom_fields: [
-                  {
-                    id: properties.contact.position.id,
-                    values: [{value: job_position}]
-                  },
-                  {
-                    id: properties.contact.phone.id,
-                    values: [{value: phone, enum: "MOB"}]
-                  },
-                  {
-                    id: properties.contact.email.id,
-                    values: [{value: email, enum: "WORK"}]
-                  }
-                ]
-              }
-            ]
+        id: id,
+        name: name,
+        date_create: to_timestamp(date_create),
+        last_modified: to_timestamp(last_modified) || Time.now.to_i,
+        request_id: request_id,
+        responsible_user_id: responsible_user_id,
+        company_name: company_name,
+        linked_leads_id: [linked_leads_id],
+        custom_fields: [
+          {
+            id: properties.contact.position.id,
+            values: [{ value: job_position }]
+          },
+          {
+            id: properties.contact.phone.id,
+            values: [{ value: phone, enum: 'MOB' }]
+          },
+          {
+            id: properties.contact.email.id,
+            values: [{ value: email, enum: 'WORK' }]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.